src/spot_skills/src/spot_skills/mesh_utilities.py

```diff
@@ -0,0 +1,100 @@
+"""Define utility functions to compute mesh properties (e.g., subframes)."""
+
+from __future__ import annotations
+
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+import numpy as np
+import trimesh
+
+if TYPE_CHECKING:
+    Coordinate3D = tuple[float, float, float]
+
+
+def print_geometric_info(name: str, mesh: trimesh.Trimesh) -> None:
+    """Print geometric information about the given mesh to aid debugging."""
+    print(f"\nInfo about the '{name}' Mesh:")
+    print(f"Mesh units: {mesh.units}")
+
+    (min_x, min_y, min_z), (max_x, max_y, max_z) = mesh.bounds
+    print(f"Minimum XYZ coordinates: {min_x:.2f} {min_y:.2f} {min_z:.2f}")
+    print(f"Maximum XYZ coordinates: {max_x:.2f} {max_y:.2f} {max_z:.2f}")
+
+    x_size = max_x - min_x
+    y_size = max_y - min_y
+    z_size = max_z - min_z
+    print(f"Size in x-coordinate: {x_size:.2f}")
+    print(f"Size in y-coordinate: {y_size:.2f}")
+    print(f"Size in z-coordinate: {z_size:.2f}")
+
+    print()
+
+
+def load_normalized_mesh(
+    mesh_path: str,
+    adjustment_matrix: np.ndarray | None = None,
+) -> trimesh.Trimesh:
+    """Load a mesh from file and transform it to a normalized frame.
+
+    The mesh is transformed to be centered in X/Y and have a minimum Z of zero.
+
+    :param mesh_path: Path to the mesh file to be imported
+    :param adjustment_matrix: Transform applied to the mesh after it's normalized
+
+    :returns: Trimesh object transformed to a normalized and axis-aligned frame.
+    """
+    geometry = trimesh.load_mesh(mesh_path)
+
+    # Identify the type of the loaded geometry (probably a Trimesh or Scene)
+    if isinstance(geometry, trimesh.Trimesh):
+        mesh = geometry
+    elif isinstance(geometry, trimesh.Scene):
+        # geometry.show()  # Visualize the mesh; helps debugging
+        mesh = geometry.to_mesh()
+    else:
+        raise TypeError(f"{mesh_path} geometry had unexpected type: {type(geometry)}")
```

Comment:
Use `trimesh.load` with the `force="mesh"` keyword ([Reference](https://trimesh.org/trimesh.html#trimesh.load)).